vendor/sigs.k8s.io/cluster-api-provider-openstack/pkg/apis/openstackproviderconfig/v1alpha1/register.go

```diff
@@ -33,7 +33,7 @@ const GroupName = "openstackproviderconfig"
 
 var (
 	// SchemeGroupVersion is group version used to register these objects
-	SchemeGroupVersion = schema.GroupVersion{Group: "openstackproviderconfig.k8s.io", Version: "v1alpha1"}
+	SchemeGroupVersion = schema.GroupVersion{Group: "openstackproviderconfig.openshift.io", Version: "v1alpha1"}
```

Comment:
can't we use this field in the package when creating the provider in https://github.com/openshift/installer/pull/2144/files#diff-a2bd450b1cd3199485e614928c597a91 rather than duplicating it ...

Reply:
Sure! I'll propose an update soon.